acstools/acs_destripe_plus.py

```diff
@@ -18,20 +18,14 @@
 Examples
 --------
 
-In Python without TEAL (recommended):
+In Python (recommended):
```

Comment:
Do we need the "(recommended)" part? With the removal of TEAL, there is no other options from within acstools and so this is redundant.

Reply:
I think we do because there is also the CLI option?

Reply:
We only ever mention running `calacs.e` on the RTD pages and the ACS DataHandbook always references the `acstools` interface. If you're tied to it, I'm fine with leaving it but it just seems weird to say something is recommended but never mention the other option that its recommended over.

Reply:
Haha... okie dokie. Good point. I'll remove it.